File: python/misc/minutes_to_seconds.py
#!/usr/bin/env python3

"""Minutes to Seconds.

Write a function that takes an integer minutes and converts it to seconds.

Source:
https://edabit.com/challenge/FQyaaJx7orS7tiwz8
"""


def how_many_seconds(mins: int) -> int:
    """Convert minutes to seconds."""
    return mins * 60


def main():
    """Run sample how_many_seconds functions. Do not import."""
    assert how_many_seconds(6) == 360
    assert how_many_seconds(4) == 240
    assert how_many_seconds(8) == 480
    assert how_many_seconds(60) == 3600
    print('Passed.')


if __name__ == "__main__":
    main()

File: javascript/edabit/minutes_to_seconds.js
/*
Minutes to Seconds

Write a function that takes an integer minutes and converts it to seconds.

Source:
https: //edabit.com/challenge/8q54MKnRrm89pSLmW
*/

function convert(minutes) {
    return minutes * 60
}

/**
 * Run sample convert functions, print results to console.log.
 */
console.log(convert(6) == 360)
console.log(convert(4) == 240)
console.log(convert(8) == 480)
console.log(convert(60) == 3600)